InnerEye/ML/lightning_metrics.py

```diff
@@ -139,7 +139,12 @@ def _get_preds_and_targets(self) -> Tuple[torch.Tensor, torch.Tensor]:
         """
         Gets a tuple of (concatenated predictions, concatenated targets).
         """
-        return torch.cat(self.preds), torch.cat(self.targets)  # type: ignore
+        preds, targets = torch.cat(self.preds), torch.cat(self.targets)  # type: ignore
+        if preds.dim() == 2 and preds.shape[1] == 2:
+            assert preds.shape[0] == targets.shape[0]
+            return preds[:, 1], targets
+        assert preds.dim() == targets.dim() == 1 and preds.shape[0] == targets.shape[0]
+        return preds, targets
```

Comment:
Can you add comments and docstrings about the case you are handling here?

Reply:
done